userspace/libscap/scap.c

```diff
@@ -613,7 +628,7 @@ scap_t* scap_open_plugin_int(char *error, int32_t *rc, scap_open_args* oargs)
 	//
 	// Allocate the handle
 	//
-	handle = (scap_t*)malloc(sizeof(scap_t));
+	handle = (scap_t*)calloc(sizeof(scap_t), 1);
```

Comment:
this should be the standard way to call it, or am I missing something? :thinking: 
```suggestion
	handle = (scap_t*)calloc(1, sizeof(scap_t));
```

Reply:
it doesn't change anything but seems strange in this way :rofl:

Reply:
the man says `void *calloc(size_t nmemb, size_t size);` but i wouldn't change it, just strange to me :smile: